src/libraries/Common/tests/System/Net/EnterpriseTests/setup/apacheweb/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM httpd:2.4
+
+COPY ./common/krb5.conf /etc/
+
+WORKDIR /setup
+COPY ./apacheweb/*.sh ./
+RUN chmod +x *.sh
+
+# Prevents dialog prompting when installing packages
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install Kerberos client, apache Negotiate auth plugin, and diagnostics
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends libapache2-mod-auth-kerb procps krb5-user iputils-ping dnsutils nano
+
+# Copy apache2 kerb module to the right place since the apt-get install puts it in the wrong place for this docker image
+RUN cp /usr/lib/apache2/modules/mod_auth_kerb.so /usr/local/apache2/modules
```

Comment:
`ln -s src dst` can work as well. and that preserves linkage to the source.

Reply:
Thanks. Fixed.